migration/src/spreadsheets.rs

```diff
@@ -130,6 +132,68 @@ impl SheetResult {
         })
     }
 
+    /// Parse this sheet as the collection index. Updates a Collection 
+    /// which contains two Chapter Groups, each of which 
+    /// has a list of associated Chapters.
+    pub fn into_collection_index(self,
+        self_title: &String, 
+        self_wordpress_menu_id: &i64) -> Result<Collection> {
+        let mut chapters = Vec::new();
+        let mut self_intro_chapters = Vec::new();
+        for row in self.values.into_iter().skip(2){
+            if row[0].is_empty(){
+                self_intro_chapters = chapters.clone();
+                chapters = Vec::<Chapter>::new();
+            }
+            else {
+                let index_in_parent_string = row[0].clone();
+                let index_i64 = index_in_parent_string.parse::<i64>().unwrap();
+
+                let document_if_applicable = row[3].clone();
+                let mut doc_string= None;
+
+                let wp_id_if_applicable = row[5].clone();
+                let mut wp_id = None;
+
+                if wp_id_if_applicable.is_empty(){
```

Comment:
This is an anti-pattern in rust. if-elses are expressions, so you can write:
```rs
let wp_id = if ... { x } else { y };
```
This also applies to `doc_string`

Reply:
Thanks! I realized I didn't even need the if-else statements, so I just rewrote the code without them, but I will keep this in mind for the future.